base/docs/helpdb/Base.jl

```diff
@@ -923,13 +923,23 @@ Stacktrace:
 BoundsError
 
 """
-    invoke(f, types <: Tuple, args...)
+    invoke(function, ArgumentTypes::Type, arguments...)
```

Comment:
> Also, `::Type` should be `::Tuple`?
> No since, e.g., `Tuple{Int}` is a tuple type not a tuple

Is there no more specific name in the type hierarchy only for tuple types?